src/program/snabbvmx/lwaftr/setup.lua

```diff
@@ -17,6 +17,7 @@ local nh_fwd = require("apps.lwaftr.nh_fwd")
 local pci = require("lib.hardware.pci")
```

Comment:
This is the setup for SnabbVMX and is different than the normal lwaftr. The setup understands a SnabbVMX config file and makes decisions based on it. It also instantiates components that are not present in lwaftr's setup (V4V6, nh_fwd). I don't think this could be done by extending lwaftr's setup as it's a brand-new setup.

What perhaps could be possible to do is to refactor the `lwaftr_app` function in this file and make it do only the logical lwaftr setup, similarly to what is done in the `lwaftr_app()` function in `src/program/lwaftr/setup.lua`. That function is used from various entry points: `load_phy`, `load_check`, etc.

The problem with the current `lwaftr_app` function is that it does the physical setup too and it would be necessary to move that out in order to reuse it from a `lwaftr_check` function. It would be necessary as well to silent the logging messages.

Considering all this, I wrote the `lwaftr_app_check` from scratch, which is analogous to the current `lwaftr_app` function, but removing physical device initialization and the logging messages. Other parts are simplified too.